Makefile

```diff
@@ -4,9 +4,11 @@ build:
 	@cargo build --verbose
 
 test-fast:
+	@./ci/keygen.sh node-engine-io-secure 127.0.0.1
 	@cargo test --verbose --package rust_socketio --lib -- engineio::packet && cargo test --verbose --package rust_socketio --lib -- socketio::packet
 
 test-all:
+	@./ci/keygen.sh node-engine-io-secure 127.0.0.1
```

Comment:
Maybe we could add a proper make target for the keys. Then the whole script is never called when the keys are present. The make target could then be invoked by the Github action instead of invoking it directly.

Reply:
done.